### DISCUSSION TITLE
过敏、烧烤和日料指数 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#147] [General]
# 过敏、烧烤和日料指数 - Miao Yu | 于淼

https://yufree.cn/cn/2023/05/02/zibo-barbecue/

**Comment by earfanfan** [comment]
三水啊，人均三百是咋估出来的，就是你突然灵机一动么？

我查了一下我老家只有一家日料店，消费水平在人均一百。又搜了一下，我老家的 GDP 在850亿这个量级，常住人口83万，淄博的 GDP 在4400亿这个量级，常住人口470万，简单一除约是人均创造9-10万GDP。虽然我长这么大都没吃过一顿人均三百的饭，但是猜想我老家应该也存在提供消费超过人均三百的场所，毕竟在财力金字塔尖尖的人肯定是哪哪都存在的。你这日料指数，应该说高端餐饮指数，仅限于日料就有点坑了吧。

**Comment by yufree** [comment]
主要因为大众点评应用里最高价位档就是人均300。我选日料就是看重其必须是外来户当地人吃的特性，而且高端日料也都不是连锁，对食材冷链都要求很高，甚至大都需要日本食材且要熟客介绍来维系，只有这个城市常住人口形成稳定高端美食需求或引进外资才会开，不然开了也得倒闭。当地有超过300的餐馆我并不意外，甚至再高都正常，本地高端餐厅大概率经营当地特色餐饮让外地人消费的，而且当地有消费能力的也大概率可以选择去一线准一线城市吃高端日料，能在当地维持人均300的日料其实很说明当地自身要具备吸引人口的潜质。另外我也没说开封菜金拱门这些连锁，他们确实也对应本地需求，但经营成本比日料低很多，选址也在流动人口多的地方，不好反应当地真实需求。
